ray-curator/ray_curator/stages/services/openai_client.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import warnings
```

Comment:
Could we also remove the equivalent scripts from the old `nemo_curator` folders wherever the new files/code covers everything from those old modules?

Reply:
True, although I only kept `heuristic_filter.py` in the `nemo_curator/` directory to keep the `LengthRatioFilter(BitextFilter)` which does not exist for Ray yet. The remaining filters were all deleted from `nemo_curator/` and moved to `ray_curator/`.

Since these files directly match to the ones in `nemo_curator/services/`, I think we should delete `nemo_curator/services/` in this PR.